libcontainer/state_linux.go

```diff
@@ -170,14 +170,13 @@ func (p *pausedState) transition(s containerState) error {
 }
 
 func (p *pausedState) destroy() error {
-	t := p.c.runType()
-	if t != Running && t != Created {
-		if err := p.c.cgroupManager.Freeze(configs.Thawed); err != nil {
-			return err
-		}
-		return destroy(p.c)
+	if p.c.hasInit() {
+		return ErrPaused
```

Comment:
when `t == Created`:

* Before this commit: It returns ErrPaused.
* After this commit: `p.c.cgroupManager.Freeze(configs.Thawed)` is executed.